src/status_im2/subs/activity_center_test.cljs

```diff
@@ -64,27 +63,3 @@
      types/admin                7})
 
   (is (= 28 (rf/sub [sub-name]))))
-
-(h/deftest-sub :activity-center/pending-contact-requests
```

Comment:
yes, that's how we should deal with tests :)

Reply:
Oh man, you have no idea how much time the unit tests for event handlers saved me from introducing bugs. But yeah, this one can go away with the simplified app db.